app/assets/javascripts/discourse/app/components/form-template-field/upload.js

```diff
@@ -0,0 +1,69 @@
+import Component from "@ember/component";
+import UppyUploadMixin from "discourse/mixins/uppy-upload";
+import { computed } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { dasherize } from "@ember/string";
+import { isAudio, isImage, isVideo } from "discourse/lib/uploads";
+
+export default class FormTemplateFieldUpload extends Component.extend(
+  UppyUploadMixin
+) {
+  @tracked uploadValue;
+  @tracked fileUploadElementId = this.attributes?.label
+    ? `${dasherize(this.attributes.label)}-uploader`
+    : `${this.elementId}-uploader`;
+  @tracked fileInputSelector = `#${this.fileUploadElementId}`;
+  @tracked id = this.fileUploadElementId;
```

Comment:
It doesn't seem like this is being used too and even if it is, I'm wondering why it has to be a tracked property when `this.fileUploadElementId` is already a tracked property.